src/providers/VoiceFocusProvider/docs/useVoiceFocus.stories.mdx

```diff
@@ -51,7 +53,7 @@ const MyChild = () => {
   const meetingManager = useMeetingManager();
   const [isVoiceFocusOn, setIsVoiceFocusOn] = useState(false);
   const [isVoiceFocusEnabled, setIsVoiceFocusEnabled] = useState(false);
-  const [device, setDevice] = useState(meetingManager.selectedAudioInputTransformDevice)
+  const [device, setDevice] = useState(meetingManager.selectedAudioInputDevice);
```

Comment:
We check for `device instanceof VoiceFocusTransformDevice` so should this be `meetingManager.selectedAudioInputDevice` or `meetingManager.selectedAudioInputTransformDevice`?

Reply:
We don't check for `device instanceof VoiceFocusTransformDevice` in this doc. It would be better to set the default value as `meetingManager.selectedAudioInputDevice` since users may set audio input before they enter the meeting like our meeting demo, and it would only update `meetingManager.selectedAudioInputDevice`. 

It was `meetingManager.selectedAudioInputTransformDevice` before and QA found the bug, so we changed it to `meetingManager.selectedAudioInputDevice` in the meeting demo.